packages/grafana-data/src/types/featureToggles.gen.ts

```diff
@@ -18,6 +18,7 @@ export interface FeatureToggles {
 
   trimDefaults?: boolean;
   envelopeEncryption?: boolean;
```

Comment:
Should this be removed then? 🤔

Reply:
I think depending on https://github.com/grafana/grafana/pull/49301#discussion_r878069404 haha
Any particular thought?

Reply:
Sorry, I didn't see that comment when leaving my comment 😆 
I do not know why we would need to leave it in place. Maybe someone else can come up with such a case - let's maybe wait for others folks' opinions.